utils/build/build.sh

```diff
@@ -155,9 +155,11 @@ build() {
                     fi
                 fi
                 source venv/bin/activate
-                python -m pip install --upgrade pip wheel
+                python -m pip install --upgrade pip setuptools==75.8.0
```

Comment:
setuptools==75.8.0  must be precised in the requirements.txt. Otherwise we may have some corrupted `venv`